_includes/templates/forms/plain-writing-feedback-2-form.html

```diff
@@ -0,0 +1,162 @@
+    <section class="block
+                    block__bg
+                    block__border
+                    block__flush-sides">
+        <form class="form-l js-validate-filters">
+            <div class="form-l_col form-l_col-1-2">
+                <label class="form-label-header" for="plain-writing-fb_first-name">
+                    First name
+                </label>
+                <input class="input__long"
+                       id="plain-writing-fb_first-name"
+                       required
+                       type="text">
+            </div>
+
+            <div class="form-l_col form-l_col-1-2">
+                <label class="form-label-header" for="plain-writing-fb_last-name">
+                    Last name
+                </label>
+                <input class="input__long"
+                       id="plain-writing-fb_last-name"
+                       required
+                       type="text">
+            </div>
+
+            <div class="form-l_col form-l_col-1">
+                <label class="form-label-header"
+                       for="plain-writing-fb_organization-address">
+                    Address
+                </label>
+                <input class="input__long"
+                       id="plain-writing-fb_organization-address"
+                       type="text"
+                       required>
+            </div>
+
+            <div class="form-l_col form-l_col-1-2">
+                <label class="form-label-header"
+                       for="plain-writing-fb_organization-city">
+                    City
+                </label>
+                <input class="input__long"
+                       id="plain-writing-fb_organization-city"
+                       type="text"
+                       required>
+            </div>
+
+            <div class="form-l_col form-l_col-1-2">
+                <div class="form-l_col form-l_col-1-2">
+                    <label class="form-label-header"
+                           for="plain-writing-fb_organization-state">
+                        State
+                    </label>
+                    <div class="custom-select">
+                        <select class="custom-select_select"
+                                data-placeholder="State"
+                                id="plain-writing-fb_organization-state"
+                                required>
+                            <option value="">Please select</option>
+                            <option value="AL">Alabama</option>
+                            <option value="AK">Alaska</option>
+                            <option value="AZ">Arizona</option>
+                            <option value="Other">Other States to be added</option>
```

Comment:
Yep, Im working on something for reusable dom components using http://www.crummy.com/software/BeautifulSoup/